package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Polyfill of future proposal for `util.parseArgs()`",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node test/index.js"
```

Comment:
For test coverage, which I think will be helpful for writing a parser, I suggest we try [c8](https://github.com/bcoe/c8) which is what I've got the Node.js using 😄 

Also, I think it would be nice to pull in `standard` or `standardx` as a linter, I usually do this:

```
"posttest": "standard",
"fix": "standard --fix"
```

That way someone can just run `npm run fix`, to address linting issues.

Reply:
let's please just use eslint, regardless of how it's configured; that package's name doesn't need any additional artificial inflation of its standard-ness.